### ISSUE TITLE
Fail to reproduce the MIC results

### ISSUE THREAD

**Issue by HuangChiEn** [lhoyer/MIC#86]
Hello, we want to benchmark with MIC in our work, while we also encountered several issue in reproduction the paper results (`75.9 mIOU` in GTA->cityscape). 

We have review all previous threads https://github.com/lhoyer/MIC/issues/8, https://github.com/lhoyer/MIC/issues/9 about the reproduced issue. 

To be honestly, we don't want to stick on the specific package version of mmcv or torch. Because the specific version of package one day will be deprecated away, the method and framework design itself should be robust across various version (the small variance of performance (<1~2% mIOU) could be acceptable). If the performance variance out of 2% range, it may show the bad case happened in a very bad way (hoeffding inequality).

---

Info field : 
The following log shows the reproduced results only achieved `70.24 mIOU` (6% dropping is huge)...orz.
[20241018_065618.log](https://github.com/user-attachments/files/17473407/20241018_065618.log)

config : 
[241018_0655_gtaHR2csHR_mic_hrda_s2_baseline_29a2d.txt](https://github.com/user-attachments/files/17473545/241018_0655_gtaHR2csHR_mic_hrda_s2_baseline_29a2d.txt)

env : 
[mic_env.txt](https://github.com/user-attachments/files/17473535/mic_env.txt)

---

Finally, we're in the progress of running the code with fixed `seed=1`.
Hope the performance difference can be mitigated by this setup.

Any suggestion will be appreciated!!

**Comment by zmyhh** [comment]
> 你好，我们希望在工作中与MIC进行基准测试，同时在复制论文结果时也遇到了一些问题（在GTA-> cityscape中）。`75.9 mIOU`
> 
> 我们已经审查了之前关于重现问题的所有帖子#[8](https://github.com/lhoyer/MIC/issues/8)和[#9](https://github.com/lhoyer/MIC/issues/9)。
> 
> 说实话，我们不想拘泥于MMCV或Torch的具体版本。由于某个特定版本的软件包终有一天会被弃用，方法和框架设计本身应在不同版本间保持鲁棒性（性能的微小差异（<1~2% mIOU）可以接受）。如果性能方差超过2%，可能说明坏情况发生在非常糟糕的情况下（不平等严重化）。
> 
> 信息字段：以下日志显示仅实现的复现结果（6%下降幅度很大）......奥兹。 [20241018_065618.log](https://github.com/user-attachments/files/17473407/20241018_065618.log)`70.24 mIOU`
> 
> 配置：[241018_0655_gtaHR2csHR_mic_hrda_s2_baseline_29a2d.txt](https://github.com/user-attachments/files/17473545/241018_0655_gtaHR2csHR_mic_hrda_s2_baseline_29a2d.txt)
> 
> 环境：[mic_env.txt](https://github.com/user-attachments/files/17473535/mic_env.txt)
> 
> 最后，我们正在运行带有固定 . 的代码。希望这种设置能弥补性能差异。`seed=1`
> 
> 任何建议都非常感谢！！

Hello, I’m very sorry to bother you. Since the official link is no longer active, could you please provide the pre-trained weights for MIT? I’m not sure if it’s an issue with the weights, but I haven’t been able to reproduce the official results. Thank you very much.
